ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java

```diff
@@ -535,6 +613,43 @@ private void waitForWarmStateStore(
     LOG.info("Timed out waiting for correct response");
   }
 
+  private void waitForRunningPersistentQueries(
+  ) {
+    final long threshold = System.currentTimeMillis() + MAX_STATIC_WARM_UP.toMillis();
+    while (System.currentTimeMillis() < threshold) {
+      final RestResponse<KsqlEntityList> resp;
+      try {
+        resp = restClient.makeKsqlRequest("SHOW QUERIES;");
+      } catch (Throwable t) {
+        t.printStackTrace();
+        throw t;
+      }
+      if (resp.isErroneous()) {
```

Comment:
nit: we can move this out of the while loop now, right? (and the error below would no longer be accurate)